addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -79,9 +79,20 @@
         @include media-only(screen) {
             --ControlPanel-border-bottom: none;
         }
+    }
 
-        &.o_field_x2many_kanban .o_kanban_ghost {
-            display: none;
+    &.o_field_x2many_kanban .o_kanban_ghost {
+        display: none;
+    }
+
+    &.o_kanban_full_width > .o_kanban_ungrouped {
```

Comment:
I checked a while ago with @aab-odoo and he did not like the generic implementation of the all-width kanban cards.
Maybe he changed his mind ?

Reply:
indeed, that's against the principles we established with the new kanban (no more magic classnames)